### ISSUE TITLE
Scroll not working on android device

### ISSUE THREAD

**Issue by dov11** [klarna/kco-mobile-sdk#33]
Hello,
We are using the Klarna sdk in a reac-native app using the wrapper [react-native-klarna](https://github.com/lightbasenl/react-native-klarna). It was working fine until recently, when we noticed that scrolling of Klarna checkout view stopped working on android.

I have made a simple native android app to test if the problem exists for native apps.
It can be found [here](https://github.com/dov11/testklarnanative). 

In this app the scrolling works on emulator, but not on a device (tested on Galaxy S7 and Nexus 5X). You can see example videos in the repo.

Scrolling can be made to work if instead of RelativeLayout, ScrollView is used [here](https://github.com/dov11/testklarnanative/blob/98d22122066444a9f473134f264b08f0cc679f8b/app/src/main/res/layout/activity_main.xml#L9). 
However this is against the recommendation in the sdk [README](https://github.com/klarna/kco-mobile-sdk/tree/master/android#presenting-the-checkout-view) as this might introduce bugs.
And we indeed see bugs arising when trying to use scrollview as a wrapper in our react-native implementation (often scrolling is limited, and cannot reach the end).

**Comment by adrianpalmquist** [comment]
Hi, thank you for reaching out. We have [updated the README](https://github.com/klarna/kco-mobile-sdk/tree/master/android#presenting-the-checkout-view) with a new recommendation which is to place a `ScrollView` somewhere in the parent hierarchy of the payment container that you create to enable scrolling on all devices.

If you still experience problems with scrolling and it's related to #27 please let us know below. If it's different from that issue please create a new issue and we'll investigate.

**Comment by dov11** [comment]
Thanks for your reply, I am now placing my Klarna view like so (programmatically generated):
```
<ScrollView>
   <LinearLayout>
       <FrameLayout>
           <KlarnaView>
```
This enables scrolling, however now, under certain conditions, the view is cut-off (might be related to #27?). 
Image 1 - View is cut-off. Image 2 - same View expands correctly after interacting with an input field.

Image 1:


![Screenshot_20190710-151457_wayke-debug](https://user-images.githubusercontent.com/27692684/60972520-bc046480-a326-11e9-93c1-232b9e5ece7d.jpg)

Image 2:
![Screenshot_20190710-151529_wayke-debug](https://user-images.githubusercontent.com/27692684/60972687-0c7bc200-a327-11e9-9535-2cf7c63613be.jpg)

Please let me know if a separate issue should be opened instead.

**Comment by omarbh** [comment]
Hi @dov11 ! Really big thinks for reporting this. We are aware of this issue - the view is cut, and we are looking into it !

**Comment by rogerbf** [comment]
Hi @omarbh, are there any updates and/or timeline that you can share regarding this issue?

Thanks.

**Comment by adrianpalmquist** [comment]
@rogerbf Hi, this issue should be fixed in v 1.6.12. Please let us know if you still have issues with resizing and/or scrolling.
